frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -319,6 +321,8 @@
                 vm.isRegistrationOpen = details.is_registration_open;
                 vm.approved_by_admin = details.approved_by_admin;
                 vm.isRemoteChallenge = details.remote_evaluation;
+                vm.isStaticDatasetCodeUpload = details.is_static_dataset_code_upload;
```

Comment:
Why are we checking for static code upload challenge when this PR is showing the code upload environment log file? This should instead check that this challenge is not a static code upload challenge

Reply:
We are checking if it's not a static code upload challenge in the html files.